docs/Options.md

```diff
@@ -0,0 +1,530 @@
+| `SPACESHIP_GIT_STATUS_UNTRACKED` | `?` | Indicator for untracked changes |
+| `SPACESHIP_GIT_STATUS_ADDED` | `+` | Indicator for added changes |
+| `SPACESHIP_GIT_STATUS_MODIFIED` | `!` | Indicator for unstaged files |
+| `SPACESHIP_GIT_STATUS_RENAMED` | `»` | Indicator for renamed files |
+| `SPACESHIP_GIT_STATUS_DELETED` | `✘` | Indicator for deleted files |
+| `SPACESHIP_GIT_STATUS_STASHED` | `$` | Indicator for stashed changes |
+| `SPACESHIP_GIT_STATUS_UNMERGED` | `=` | Indicator for unmerged changes |
+| `SPACESHIP_GIT_STATUS_AHEAD` | `⇡` | Indicator for unpushed changes (ahead of remote branch) |
+| `SPACESHIP_GIT_STATUS_BEHIND` | `⇣` | Indicator for unpulled changes (behind of remote branch) |
+| `SPACESHIP_GIT_STATUS_DIVERGED` | `⇕` | Indicator for diverged chages (diverged with remote branch) |
+
+### Mercurial (`hg`)
+
+Mercurial section is consists with `hg_branch` and `hg_status` subsections. It is shown only in Mercurial repositories.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_HG_SHOW` | `true` | Show Mercurial section |
+| `SPACESHIP_HG_PREFIX` | `on ` | Prefix before Mercurial section |
+| `SPACESHIP_HG_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Mercurial section |
+| `SPACESHIP_HG_SYMBOL` | `☿ ` | Character to be shown before Mercurial section |
+
+#### Mercurial branch (`hg_branch`)
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_HG_BRANCH_SHOW` | `true` | Show Mercurial branch subsection |
+| `SPACESHIP_HG_BRANCH_PREFIX` | `$SPACESHIP_HG_SYMBOL` | Prefix before Mercurial branch subsection |
+| `SPACESHIP_HG_BRANCH_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Mercurial branch subsection |
+| `SPACESHIP_HG_BRANCH_COLOR` | `magenta` | Color of Mercurial branch subsection |
+
+#### Mercurial status (`hg_status`)
+
+Mercurial status indicators is shown only when you have dirty repository.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_HG_STATUS_SHOW` | `true` | Show Mercurial status subsection |
+| `SPACESHIP_HG_STATUS_PREFIX` | `[` | Prefix before Mercurial status subsection |
+| `SPACESHIP_HG_STATUS_SUFFIX` | `]` | Suffix after Mercurial status subsection |
+| `SPACESHIP_HG_STATUS_COLOR` | `red` | Color of Mercurial status subsection |
+| `SPACESHIP_HG_STATUS_UNTRACKED` | `?` | Indicator for untracked changes |
+| `SPACESHIP_HG_STATUS_ADDED` | `+` | Indicator for added changes |
+| `SPACESHIP_HG_STATUS_MODIFIED` | `!` | Indicator for unstaged files |
+| `SPACESHIP_HG_STATUS_DELETED` | `✘` | Indicator for deleted files |
+
+### Package version (`package`)
+
+> Works only for npm at the moment. Please, help us improve this section!
+
+Package version is shown when repository is a package (contains a `package.json` file). This is the version of the package you are working on, not the version of package manager itself.
```

Comment:
Since it will support not only npm packages, I'd spell it like this:

... (**e.g.** contains a `package.json file). **Note:** this is the version...

Reply:
Fixed.